dist/trumbowyg.js

```diff
@@ -653,6 +653,12 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
                         }
                     }
                 })
```

Comment:
You should merge drop and cut listener since they have the same behavior:

```
.on('cut drop', ...
```

Reply:
Will do sir.